lib/rihanna/job.ex

```diff
@@ -87,6 +88,14 @@ defmodule Rihanna.Job do
     GenServer.call(Rihanna.JobManager, job)
   end
 
+  @doc """
+  The priority of this job.
+
+  Conforms to the niceness values used in Linux processes — lower values
+  are more important.
```

Comment:
I find the comparison to nice more confusing than clear. I would prefer to match the semantics of other work queue systems such as AMQP where a higher number means a higher priority.

Perhaps this is just my bias.

Reply:
Agree with Louis, let's drop the comparison to linux niceness I think that creates more confusion than it solves.

Higher number = higher priority works for me.

Reply:
Would like to see some clearer documentation here. I.e. is it required/optional, what is the default, what does "more important" actually mean in practice?

Reply:
What's the preferred default here?

Reply:
I've opted to go with a default of zero.